code/datums/components/footstep.dm

```diff
@@ -87,7 +87,7 @@
 		return
 	//SANITY CHECK, WILL NOT PLAY A SOUND IF THE LIST IS INVALID
 	if(!footstep_sounds[turf_footstep] || (LAZYLEN(footstep_sounds) < 3))
-		testing("SOME RETARD GAVE AN INVALID FOOTSTEP [footstep_type] VALUE ([turf_footstep]) TO [T.type]!!! FIX THIS SHIT!!!")
+		testing("SOME MUPPET GAVE AN INVALID FOOTSTEP [footstep_type] VALUE ([turf_footstep]) TO [T.type]!!! FIX THIS SHIT!!!")
```

Comment:
```suggestion
		CRASH("Invalid footstep value given. Turf type: [T.type]; Footstep Type: [footstep_type]; Value: [turf_footstep]")
```

Reply:
Yeah, let's go with your thing, seems more useful and well worded.